### ISSUE TITLE
does kcgi have any built-in support for csrf_tokens?

### ISSUE THREAD

**Issue by chrisbare** [kristapsdz/kcgi#112]
I was just wondering if checking the token might be part of form processing, or if I have to do it all in my application.

**Comment by kristapsdz** [comment]
No: CSRF protection is orthogonal to kcgi, as you'd need some manner of persistent storage.  If you're not scaling across machines, it's pretty easy and performant to just have an sqlite database (or other shared memory system) and manage tokens there.
